test/examples/ucp_hello_world.c

```diff
@@ -235,6 +236,7 @@ static int run_ucx_client(ucp_worker_h ucp_worker)
         goto err_ep;
     } else {
         wait(ucp_worker, request);
+        request->completed = 0;
```

Comment:
I think it is better to move it into wait function to reset to zero any request after wait.